src/triage/component/architect/feature_generators.py

```diff
@@ -351,6 +352,46 @@ def generate_all_table_tasks(self, aggregations, task_type):
         logging.info('Created %s tables', len(table_tasks.keys()))
         return table_tasks
 
+    def create_preimputed_features(self, feature_aggregation_config, feature_dates):
```

Comment:
"preimputed" sounds to me like "already imputed" rather than "before being imputed" (like "pre-peeled garlic" or "pre-cooked chicken"). Maybe "unimputed"?